storage/plugins/archives/ArchivesPlugin.php

```
@@ -5224,6 +5271,241 @@ public function iiifCollectionAction(
+        return $this->ricJsonResponse($response, $doc, $builder->agentIri($id));
+    }
+
+    /**
+     * Build a per-request RicJsonLdBuilder configured with the current
+     * canonical base URL and locale. Kept private so callers cannot bypass
+     * the canonical-URL resolution path.
+     */
+    private function makeRicBuilder(): RicJsonLdBuilder
+    {
+        $base   = rtrim(absoluteUrl(''), '/');
+        $locale = (string) (\App\Support\I18n::getLocale() ?: 'en');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\I18n' in method 'makeRicBuilder'. (undefined)

(StaticAccess)